docs/io/ascii/index.rst

```diff
@@ -229,6 +229,63 @@ ASCII with no loss of information.
 ..
   EXAMPLE END
 
+Miscellaneous Examples
+----------------------
+Some points of interests pertaining to masking are the ``fill_include_names``
+and ``fill_exclude_names`` parameters of the :class:`~astropy.io.ascii.BaseData` class.
+The original intention for the usage of these parameters are in the ``masks``
+method of the same :class:`~astropy.io.ascii.BaseData` class. In the first step of the
+``masks`` method, the mask is evaluated with values from ``fill_values`` as applied
+to which column using ``fill_include_names`` and ``fill_exclude_names``.
+In the second step, all the replacements of values are done for the appropriate columns.
+As standalone tools, they can be used to serve in isolating columns that are of interests
+while not deleting them outright. We can use the ``fill_include_names`` parameter in include
+only columns with names as indicated. On the contrary but highly similarly, we can use
+the ``fill_exclude_names`` parameter to exclude the columns with the names as indicated.
```

Comment:
This whole section is a bit too detailed and hard to follow.  I'd suggest just boiling it down to this:
```
The |read| function provides parameters ``fill_include_names`` and ``fill_exclude_names``
to select which columns will be used in the ``fill_values`` masking process described above.
The use of these parameters is not common but in some cases can considerably simplify
the code required to read a table.
```

Reply:
Sure! I have made some changes according to the suggestions in the most recent commit(s).